Assets/Scripts/GrindingGame/CorrectKeyPress.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
```

Comment:
You should comment your name and general purpose of the class and also put comments throughout the code for readability. I know Burke likes that.